### DISCUSSION TITLE
修改 csghub 的域名后，老的 model、dataset clone 时域名没有同步更新，如何处理?

### DISCUSSION THREAD

**Discussion by karypi** [OpenCSGs/csghub#612] [Q&A]

**Comment by HaiHui886** [comment]
更新一下repository表的clone url值

**Comment by karypi** [comment]
能否提供下命令？

**Comment by HaiHui886** [comment]
先登录运行环境的postgres数据库starhub_server, 找到表respository，使用update sql语句，更新http_clone_url字段值，把旧的域名修改成新域名.

**Reply by Rader** [reply]
ssh_clone_url也需要同步修改

**Comment by karypi** [comment]
问题已解决，感谢！

**Reply by Rader** [reply]
帮忙标记一下答案，方便其他用户参考，谢谢！

**Comment by Rader** [comment]
下一个版本不会有这个问题了，可以随时修改域名，最新的代码已经在main 分支上。
